server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -332,10 +332,11 @@
      * Logs an informational message indicating the server is starting.
      *
      * @param prettyVersion  the server version.
+     * @param coolName the server cool name
      */
     @LogMessage(level = INFO)
-    @Message(id = 49, value = "%s starting")
-    void serverStarting(String prettyVersion);
+    @Message(id = 49, value = "%s starting %s")
```

Comment:
Could you please change the `value` to `"%s starting/n%s"`?

This log is very important and I want to minimise the change (so that the line still ends with `starting`)

Reply:
The new line was in the banner to make it easy to edit